### DISCUSSION TITLE
Picture (??) is missing on login screen

### DISCUSSION THREAD

**Discussion by vampywiz17** [LingmoOS/Community#13] [Bugs Report/缺陷提交] [answered]
I think in the future we will see in these place are a picture or something, but now it is emtpy:

![image](https://github.com/user-attachments/assets/756381a8-3e0e-44f6-9b29-925ddc18d626)

**Comment by elysia-best** [comment]
Yes, I've just noticed that. @lingmo-dream

**Reply by elysia-best** [reply, marked as answer]
You can simply change the icon in Settings app and it will be shown in the welcome screen.

**Reply by vampywiz17** [reply]
it would be great to set it up a default one, i think. For example these one that show in settings menu.

**Comment by elysia-best** [comment]
This should be addressed now.
